packages/kolibri-common/strings/enhancedQuizManagementStrings.js

```diff
@@ -238,28 +229,26 @@ export const enhancedQuizManagementStrings = createTranslator('EnhancedQuizManag
   },
   notEnoughReplacementsTitle: {
     message: 'Not enough replacements available',
-    context:
-      'Title of modal when a user tries to replace more questions than are available in the pool',
-  },
-  noReplacementsMessage: {
+    context: 'Title of modal when a user tries to replace more questions than are available',
+  },
+  // noReplacementsMessage: {
+  //   message:
+  //     'There are no more similar questions available to replace the selected questions with.',
+  // },
+  // notEnoughReplacementsMessage: {
+  //   message:
+  //     "You've selected { selected, number } { selected, plural, one { question } other { questions } } to replace, but {available, plural, =0 { don't have questions } one { only have 1 question } other { only have { available } questions } } available to replace them with.",
+  //   context:
+  //     'Message of modal when a user tries to replace more questions than are available in the pool',
+  // },
+  goBackAndUnselect: {
+    message: 'You can go back and unselect some questions to be able to perform the replacement.',
```

Comment:
The only problem here is that at no point now are we telling the user how many replacements there are available, so they don't know how many to select.

Reply:
There was a _lot_ of discussion here about whether or not adding the numbers was helpful, or made things more confusing. I don't know if we will reach consensus... My thought is that we pick a direction and either we revisit in 0.18 when some of the workflow changes in general, or try to get some feedback on during training/observations or from our community? (maybe it's a bit granular to expect commentary on)